pages.ja/common/docker.md

```diff
@@ -0,0 +1,37 @@
+# docker
+
+> Dockerコンテナ及びDockerイメージの管理を行います。
+> `docker run`などいくつかのサブコマンドがあります。これらには使用方法についての独自のドキュメントがあります。
+> 詳しくはこちら: <https://docs.docker.com/engine/reference/commandline/cli/>
```

Comment:
```suggestion
> 詳しくはこちら: <https://docs.docker.com/engine/reference/commandline/cli/>.
```

Reply:
Fixed in https://github.com/tldr-pages/tldr/pull/9785/commits/49628382d32889787e6a8d145afef8cfe5c695e7